src/libraries/System.Globalization/tests/IcuAppLocal/IcuAppLocal.cs

```diff
@@ -0,0 +1,48 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.DotNet.RemoteExecutor;
+using System.Diagnostics;
+using System.Reflection;
+using Xunit;
+
+namespace System.Globalization.Tests
+{
+    public class IcuAppLocalTests
+    {
+        private static bool SupportsIcuPackageDownload => RemoteExecutor.IsSupported &&
+                                                          ((PlatformDetection.IsWindows && !PlatformDetection.IsArmProcess) ||
+                                                           (PlatformDetection.IsLinux && (PlatformDetection.IsX64Process || PlatformDetection.IsArm64Process) &&
+                                                           !PlatformDetection.IsAlpine && !PlatformDetection.IsLinuxBionic));
+
+
+        [ConditionalFact(nameof(SupportsIcuPackageDownload))]
+        public void TestIcuAppLocal()
+        {
+            // We define this switch dynamically during the runtime using RemoteExecutor.
+            // The reason is, if we enable ICU app-local here, this test will compile and run
+            // on all supported OSs even the ICU NuGet package not have native bits support such OSs.
+            // Note, it doesn't matter if we have test case conditioned to not run on such OSs, because
+            // the test has to start running first before filtering the test cases and the globalization
+            // code will run and fail fast at that time.
+
+            ProcessStartInfo psi = new ProcessStartInfo() { UseShellExecute = false };
```

Comment:
```suggestion
            ProcessStartInfo psi = new ProcessStartInfo();
```
`UseShellExecute` defaults to false in .NET Core, this is unnecessary.